tests/driver_ina2xx/README.md

```diff
@@ -1,5 +1,6 @@
 # About
-This is a manual test application for the INA220 current and power monitor driver.
+This is a manual test application for the INA210/INA220 current and power
```

Comment:
nit-picky: the test app uses `ina219` according to the Makefile

Reply:
You're right!

Reply:
Also it is completely outdated :-( Good, that I look at it again